src/main/java/bio/terra/service/snapshot/SnapshotService.java

```diff
@@ -1018,6 +1048,9 @@ private SnapshotModel populateSnapshotModelFromSnapshot(
     if (include.contains(SnapshotRetrieveIncludeModel.PROPERTIES)) {
       snapshotModel.properties(snapshot.getProperties());
     }
+    if (include.contains(SnapshotRetrieveIncludeModel.DUOS)) {
```

Comment:
Note: readers can retrieve a snapshot, and while they might not find information on the snapshot's DUOS link useful, I think that they should be permitted to see it even though they can't set it because DUOS datasets are publicly discoverable to authed users.

However, we may elect not to retrieve this or render it in the TDR UI for snapshot readers.